content/open-source/octo-sts/faq.md

```diff
@@ -0,0 +1,379 @@
+
+### Why should I use Octo STS instead of Personal Access Tokens?
+
+Personal Access Tokens are long-lived credentials that pose security risks:
+- They don't expire automatically
+- They're often stored in multiple places
+- If leaked, they provide persistent access
+- They're difficult to rotate across many systems
+
+Octo STS provides:
+- Short-lived tokens that expire automatically
+- No credentials to store in your automation
+- Audit trail of token usage
+- Fine-grained permission control per workload
+
+### How does Octo STS compare to GitHub's built-in GITHUB_TOKEN?
+
+GitHub Actions provides a `GITHUB_TOKEN` automatically, but it has limitations:
+- Cannot update workflow files
+- Cannot trigger other workflows
+- Limited to a fixed set of permissions
+
+Octo STS tokens can:
+- Update workflow files (useful for Renovate)
+- Have any permissions defined in your trust policy
+- Work consistently across all automation platforms, not just GitHub Actions
+
+### Is Octo STS free to use?
+
+Yes, octo-sts is open source and the hosted service at octo-sts.dev is free to use. You can also self-host Octo STS if you prefer.
+
+### Can I self-host Octo STS?
+
+Yes, octo-sts is open source and can be self-hosted. See the [Octo STS repository](https://github.com/octo-sts/app) for deployment instructions.
+
+## Setup and Configuration
+
+### How do I install Octo STS?
+
+Install the GitHub App:
+1. Visit [https://github.com/apps/octo-sts](https://github.com/apps/octo-sts)
+2. Click **Install**
+3. Select the organization or user account
+4. Choose which repositories to grant access
+5. Approve the permissions
+
+Then create trust policies in your repositories at `.github/chainguard/{name}.sts.yaml`.
+
+### Why does Octo STS request so many permissions?
+
+Octo STS requests a superset of permissions to support all possible use cases. However, it only creates tokens with the specific permissions defined in your trust policies. The app needs `contents: read` to read trust policy files, but all other permissions are only granted based on your policies.
+
+### What happens if I don't create any trust policies?
+
+If you install Octo STS but don't create trust policies, the app cannot issue any tokens. Trust policies are required to specify which identities are trusted and what permissions to grant them.
+
+### Can I use octo-sts with private repositories?
+
+Yes, octo-sts works with both public and private repositories. The app needs access to read the repository's trust policy files.
+
+### How do I update trust policy permissions?
+
+Edit the trust policy file in your repository, commit, and push the changes. The new permissions take effect immediately for subsequent token exchanges. Existing tokens retain their original permissions until they expire.
```

Comment:
Have we said anything about expiration time?

Reply:
Dealt with this (I clicked "don't ask me again" and now I can't delete anything)